specification/resources/apps/models/app_route_spec.yml

```
@@ -6,3 +6,7 @@ properties:
       all components within an app.
     type: string
     example: /api
+  preserve_path_prefix:
+    description:  Boolean. An optional flag to preserve the path that is forwarded to the backend service. By default, the HTTP request path will be trimmed from the left when forwarded to the component. For example, a component with `path=/api` will have requests to `/api/list` trimmed to `/list`. If this value is `true`, the path will remain `/api/list`.
```

Does `Boolean` need to be specified in the description if the property has `type: boolean`?

```suggestion
    description: An optional flag to preserve the path that is forwarded to the backend service. By default, the HTTP request path will be trimmed from the left when forwarded to the component. For example, a component with `path=/api` will have requests to `/api/list` trimmed to `/list`. If this value is `true`, the path will remain `/api/list`.
```

---

Sorry, just saw this now.